tests/utils/test_scatter.py

```diff
@@ -1,6 +1,7 @@
 import mlx.core as mx
+import pytest
 
-from mlx_graphs.utils import scatter
+from mlx_graphs.utils import scatter, degree
 
 
 def test_scatter():
```

Comment:
You can put `test_scatter` and `test_scatter_2D` together and parametrize the test